index.php

```diff
@@ -50,11 +50,16 @@
         ]
     ],
     'settings' => '@flick',
+	
+   'resources' => [
+        'sgerli/flick:' => '',
+    ],
+
     'events' => [
         'view.scripts' => function ($event, $scripts) use ($app) {
 
-			$scripts->register('uikit-grid', 'app/assets/uikit/js/components/grid.min.js', 'uikit');
-			$scripts->register('uikit-lightbox', 'app/assets/uikit/js/components/lightbox.min.js', 'uikit');
+			$scripts->register('uikit-grid', 'sgerli/flick:app/assets/uikit/js/components/grid.min.js', 'uikit');
+			$scripts->register('uikit-lightbox', 'sgerli/flick:app/assets/uikit/js/components/lightbox.min.js', 'uikit');
```

Comment:
This two (61 and 62) lines don't work with `sgerli/flick:` because they require files from Pagekit, not the extension.

Reply:
Wouldn't it be better to include those like this:

`$scripts->register('some-stuff', 'sgerli/flick:app/assets/my.css', ['uikit', 'uikit-grid', 'uikit-lightbox']);`

?

Reply:
That's already in `views/index.php` line 7, and we need to register those dependencies.

Reply:
After a bit more testing I realized we can completely remove this two lines (61, 62) and it still loads them as dependencies, but I still have the sort problem.

Reply:
So now just waiting for rebase with lines 18 and 53 from `FlickController.php` fixed.